src/OrchardCore.Modules/OrchardCore.GitHub/Configuration/GithubHandler.cs

```diff
@@ -16,9 +16,16 @@ namespace OrchardCore.GitHub.Configuration
 {
     public class GitHubHandler : OAuthHandler<GitHubOptions>
     {
+
+#if !NET8_0_OR_GREATER
         public GitHubHandler(IOptionsMonitor<GitHubOptions> options, ILoggerFactory logger, UrlEncoder encoder, ISystemClock clock)
             : base(options, logger, encoder, clock)
         { }
+#else
+        public GitHubHandler(IOptionsMonitor<GitHubOptions> options, ILoggerFactory logger, UrlEncoder encoder)
```

Comment:
If this also works for NET7.0 then we should use this in all cases.

Reply:
Doesn’t work on net7.0